cirq/study/visualize_test.py

```diff
@@ -38,3 +38,30 @@ def test_plot_state_histogram():
     expected_values = [0., 0., 0., 5.]
 
     np.testing.assert_equal(values_plotted, expected_values)
+
+
+def test_plot_state_histogram_multi_1():
+    pl.switch_backend('PDF')
+    qubits = cirq.LineQubit.range(4)
+    c = cirq.Circuit(
+        cirq.X.on_each(*qubits[1:]),
+        cirq.measure(*qubits),  # One multi-qubit measurement
+    )
+    r = cirq.sample(c, repetitions=5)
+    values_plotted = visualize.plot_state_histogram(r)
+    expected_values = [0, 0, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, 0]
+    np.testing.assert_equal(values_plotted, expected_values)
+
+
+def test_plot_state_histogram_multi_2():
+    pl.switch_backend('PDF')
+    qubits = cirq.LineQubit.range(4)
+    c = cirq.Circuit(
+        cirq.X.on_each(*qubits[1:]),
+        cirq.measure(*qubits[:2]),  # One multi-qubit measurement
+        cirq.measure_each(*qubits[2:]),  # One single-qubit measurement
```

Comment:
Nit: The comment is wrong.  It is now two single-qubit measurements.

Reply:
Fixed.